config/packages/process/demo.rest.yaml

```diff
@@ -0,0 +1,46 @@
+clever_age_process:
+    configurations:
+        demo.rest:
+            options:
```

Comment:
Il peut être intéressant de rajouter un exemple d'exécution du flux. J'ai un peu du jouer avec le contexte pour avoir la syntaxe afin que le codePostal ne soit pas converti en int et fasse planter le process.

Ex: bin/console cleverage:p:e demo.rest -c codePostal:46800 fait planter le process avec l'erreur   Process demo.rest has failed during process entry with message: 'rawurlencode(): Argument #1 ($string) must be of type string, int given'.  alors que si on veut avoir des résultat de l'api il faut passer la commande bin/console cleverage:p:e demo.rest -c codePostal:"'46800'"

Reply:
TODO : ajouter : 
``ỳaml
description: >
                Ex: bin/console cleverage:process:execute demo.rest -c codePostal:"'46800'"
```